src/mongocxx/insert_many_builder.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2015 MongoDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <mongocxx/config/prelude.hpp>
+
+#include <mongocxx/insert_many_builder.hpp>
+
+#include <bsoncxx/builder/stream/document.hpp>
+
```

Comment:
No blank line between these headers. Order and spacing of headers is:

prelude

self-header <if applicable>

<sorted list of needed mongocxx/bsoncxx headers with no blank lines>